### PR TITLE
First draft documentation

### PR DESCRIPTION
**Pull request by sudhirunicus**
Please provide feedback and information for these documents

### CODE REVIEW

File: docs/platform/SEPES component overview.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+# SEPES #
+
+In this document SEPES, with its different components, will be described/ documented. 
+
+## SEPES Frontend ##
+
+Frontend allowes users to manage
```
Comment (Unicus-Tom) [inline docs/platform/SEPES component overview.md]: Incomplete sentence

File: docs/platform/SEPES component overview.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+Tech Stack: 
+
+Maturity: 3
+
+Protocol: https
+
+## Azure Portal ##
+
+Azure Portal is used by vendors to create resources for their experiment. 
+Documentation can be found at microsoft homepage
```
Comment (Unicus-Tom) [inline docs/platform/SEPES component overview.md]: https://docs.microsoft.com/nb-no/azure/

File: docs/platform/SEPES component overview.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+Maturity: 
+
+Protocol: https
+
+## SEPES REST API (Backend) ##
+
+Coordinate the different services, and make actions on the behalf of the user.
+
+Tech Stack: asp.net core, C#
```
Comment (Unicus-Tom) [inline docs/platform/SEPES component overview.md]: dotnet, asp.net, C#

File: docs/platform/SEPES component overview.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+## Data Source
+
+The data catalog provides the link to data.
+
+Protocol: https
+
+
+
+Maturity is based on Maufacturing Readiness Level. https://rescoll.fr/wp-content/uploads/2014/01/MRL.jpg
```
Comment (Unicus-Tom) [inline docs/platform/SEPES component overview.md]: These should be in same order as is done in other uses of maturity and protocol

File: docs/platform/SEPES component overview.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+Protocol: Uses http, so have to be driven in the docker. Link documentation at OPA's site. OPA is driven in SEPES RestAPI.
+
+
+## Azure AD ##
+
+Used for autentication
+
+Maturity: 
+
+Link for Azure AD use. https://docs.microsoft.com/en-us/azure/active-directory/
```
Comment (Unicus-Tom) [inline docs/platform/SEPES component overview.md]: Should be moved up to describtion

File: docs/platform/SEPES component overview.md (hunk 2 above)
Comment (gjersvik) [reply docs/platform/SEPES component overview.md]: https://docs.microsoft.com/azure/ Should use in international one that redirects to local version.

File: docs/platform/SEPES component overview.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+## SEPES Frontend ##
+
+Frontend allowes users to manage 
+
+This contains the following roles:
+- Admin: Access to all resources 
+- Sponsors: Responsible for sandbox management 
+- Supplier: External vendors
+
+Tech Stack:
```
Comment (gjersvik) [inline docs/platform/SEPES component overview.md:14]: - React JS https://reactjs.org/
- WebPack https://webpack.js.org/
- HTML, CSS and JavaScript

File: docs/platform/SEPES component overview.md (hunk 3 above)
Comment (gjersvik) [reply docs/platform/SEPES component overview.md]: We need the core part. .net and ASP.net is not the same as .net core and ASP.net core. At least until .net 5 comes out in a few years. Also think we should add the azure sdk as part of the tech stack.

File: docs/platform/SEPES component overview.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+
+Coordinate the different services, and make actions on the behalf of the user.
+
+Tech Stack: asp.net core, C#
+
+Maturity:
+
+Protocol: Uses https rest protocol and can be configured to use http. http should only be used for cases that requires it.
+
+As an example docker compose is configured to use http, this is still safe as the docker wrapper uses https outside.
```
Comment (gjersvik) [inline docs/platform/SEPES component overview.md]: Its only safe its behind a proxy that handle HTTPS terminations. Or in local development setting. Docker compose have nothing to do with protocol on it own. Proxy is the important keyword.

File: docs/platform/SEPES component overview.md (hunk 4 above)
Comment (gjersvik) [inline docs/platform/SEPES component overview.md]: I think we should look for another more appropriate maturity metrics. To use. After closer look Maufacturing Readiness Level do not seem appropriate.

**Review by gjersvik — APPROVED** [review]
Good first draft but there are tings that need to be looked at for a next version.

File: docs/platform/SEPES component overview.md [hunk 8]
```diff
@@ -4,7 +4,7 @@ In this document SEPES, with its different components, will be described/ docume
 
 ## SEPES Frontend ##
 
-Frontend allowes users to manage 
+Frontend allows users to manage pods.
```
Comment (gjersvik) [inline docs/platform/SEPES component overview.md]: Think we should use sandboxes and not pods.

File: docs/platform/SEPES component overview.md [hunk 9]
```diff
@@ -32,7 +32,7 @@ Protocol: https
 
 Coordinate the different services, and make actions on the behalf of the user.
 
-Tech Stack: asp.net core, C#
+Tech Stack: dot.net, asp.net core, C#
```
Comment (gjersvik) [inline docs/platform/SEPES component overview.md]: Its ".net" not dotnet or dot.net

File: docs/platform/SEPES component overview.md [hunk 10]
```diff
@@ -92,11 +92,11 @@ Protocol: Uses http, so have to be driven in the docker. Link documentation at O
 
 ## Azure AD ##
 
-Used for autentication
+Used for autentication. Link for Azure AD use. https://docs.microsoft.com/en-us/azure/active-directory/
```
Comment (gjersvik) [inline docs/platform/SEPES component overview.md]: https://docs.microsoft.com/azure/active-directory/ International link

File: docs/platform/SEPES component overview.md [hunk 11]
```diff
@@ -32,11 +35,13 @@ Protocol: https
 
 Coordinate the different services, and make actions on the behalf of the user.
 
-Tech Stack: dot.net, asp.net core, C#
+Tech Stack: .net, asp.net core, C#
```
Comment (gjersvik) [inline docs/platform/SEPES component overview.md:38]: Tech Stack: .net core, asp.net core, C#

File: docs/platform/SEPES component overview.md [hunk 12]
```diff
@@ -32,11 +35,13 @@ Protocol: https
...
 
 Maturity:
 
-Protocol: Uses https rest protocol and can be configured to use http. http should only be used for cases that requires it.
+Protocol: Uses https rest protocol and can be configured to use http. http should only be used for cases that requires it. 
+
+Its only safe its behind a proxy that handle HTTPS terminations. Or in local development setting. Docker compose have nothing to do with protocol on it own. Proxy is the important keyword.
```
Comment (gjersvik) [inline docs/platform/SEPES component overview.md]: Its only safe its behind a proxy that handle HTTPS terminations. Or in local development setting. ~~Docker compose have nothing to do with protocol on it own. Proxy is the important keyword.~~

File: docs/platform/SEPES component overview.md [hunk 13]
```diff
@@ -92,8 +97,7 @@ Protocol: Uses http, so have to be driven in the docker. Link documentation at O
 
 ## Azure AD ##
 
-Used for autentication. Link for Azure AD use. https://docs.microsoft.com/en-us/azure/active-directory/
-
+Used for autentication. Link for Azure AD use. https://docs.microsoft.com/nb-no/azure/active-directory/
```
Comment (gjersvik) [inline docs/platform/SEPES component overview.md]: Used for autentication. Link for Azure AD use. https://docs.microsoft.com/azure/active-directory/

**Comment by sudhirunicus** [comment]
new changes made

**Review by sudhirunicus — COMMENTED** [review]
check new changes

**Review by kribjo1968 — APPROVED** [review]
Looks good with OMG's corrections.

**Pull request merged**